app/models/payment.rb

```diff
@@ -0,0 +1,21 @@
+module Payment
+  class << self
+    def table_name_prefix
+      'payment_'
+    end
+
+    def write_transaction(transaction:, provider:)
```

Comment:
In the case of a Braintree transaction, what's the transaction and provider that get passed as arguments? Where is `write_transaction` called?